packages/open-api-gateway/test/resources/specs/multi.yaml

```diff
@@ -0,0 +1,11 @@
+openapi: 3.0.3
```

Comment:
Should probably update README to mention refs are supported (with an example too).

Reply:
Good shout, will do!